package/boot/uboot-tools/uboot-envtools/files/realtek

```diff
@@ -31,6 +31,15 @@ zyxel,gs1900-48)
 	ubootenv_add_mtd "u-boot-env" "0x0" "0x400" "0x10000"
 	ubootenv_add_sys_mtd "u-boot-env2" "0x0" "0x1000" "0x10000"
 	;;
+plasmacloud,psx8|\
+plasmacloud,psx10)
+	idx="$(find_mtd_index u-boot-env)"
+	[ -n "$idx" ] && \
+		ubootenv_add_uci_config "/dev/mtd$idx" "0x0" "0x10000"
+	idx2="$(find_mtd_index u-boot-env2)"
+	[ -n "$idx2" ] && \
+		ubootenv_add_uci_sys_config "/dev/mtd$idx2" "0x0" "0x1000"
```

Comment:
Use the new `ubootenv_add_sys_mtd`:
```suggestion
	ubootenv_add_sys_mtd "u-boot-env2" "0x0" "0x10000"
```

Reply:
Should most likely be dropped because it would match the default case:

```
*)
	ubootenv_add_mtd "u-boot-env" "0x0" "0x10000" "0x10000"
	ubootenv_add_sys_mtd "u-boot-env2" "0x0" "0x1000" "0x10000"
	;;
```